docs/index.rst

```diff
@@ -1,27 +1,9 @@
-sqlalchemy-redshift
-===================
+===============================================
+Welcome to sqlalchemy-redshift's documentation!
+===============================================
```

Comment:
I chose the because it was the default, and not the same as the heading in the README.rst.

I'll change it if you come up with a better one